diff --git a/src/swc-react-refresh.js b/src/swc-react-refresh.js
index f69299018e3abc25219265f45ed46120c32e80b3..66bf91841d83b715ca1844179701fc29a420301b 100644
--- a/src/swc-react-refresh.js
+++ b/src/swc-react-refresh.js
@@ -49,6 +49,9 @@ const swcReactRefresh = () => ({
             sourceMaps: true,
             jsc: {
                 target: "es2020",
+                parser: {
+                    jsx: true
+                },
                 transform: {
                     react: {
                         refresh: true,